Sources/Apodini/Semantic Model Builder/gRPC/ProtobufferBuilder/Model.swift

```diff
@@ -7,7 +7,7 @@
 
 // MARK: - Message
 
-struct Message: Equatable, Hashable {
+struct ProtoMessage: Equatable, Hashable {
```

Comment:
@moritzschuell and I agreed on not using abbreviations here: https://github.com/Apodini/Apodini/pull/43#discussion_r549125372

Please rename Service as well if this change is necessary.

Reply:
solved by 57e6975